utils/src/main/kotlin/OrtProxySelector.kt

```diff
@@ -171,9 +177,12 @@ class OrtProxySelector(private val fallback: ProxySelector? = null) : ProxySelec
     override fun select(uri: URI?): List<Proxy> {
         requireNotNull(uri)
 
-        if (proxyExcludes.any { it.isNotEmpty() && (uri.authority.endsWith(it) || uri.host.endsWith(it)) }) {
-            return NO_PROXY_LIST
-        }
+        fun URI.matches(suffix: String) = suffix.isNotEmpty() && (authority.endsWith(suffix) || host.endsWith(suffix))
```

Comment:
The function could even go out either on file level or in companion object, for readability.

Reply:
I had considered that, but actually found it to be more fitting here as this function is very narrow-scoped in its intended use, so I prefer to keep it close to the only place it's being used.